script/vsts/platforms/templates/publish.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  # artifacts:
+  # - fileName
+  # - fileDir
+  # - condition
+  # - continueOnError
```

Comment:
Is there a reason why these are commented out?

Reply:
This is the API documentation. It is not code.

Reply:
I changed the explanation in https://github.com/atom/atom/pull/21262/commits/366c3b56f6e8da86664baa6c5ef373e142e269a7